app.js

```diff
@@ -23,63 +34,144 @@ var stuff = [
   new RandomPicture('scissors', 'img/scissors.jpg'),
   new RandomPicture('shark', 'img/shark.jpg'),
   new RandomPicture('sweep', 'img/sweep.png'),
-  new RandomPicture ('tauntaun', 'img/tauntaun.jpg'),
-  new RandomPicture ('unicorn', 'img/unicorn.jpg'),
+  new RandomPicture('tauntaun', 'img/tauntaun.jpg'),
+  new RandomPicture('unicorn', 'img/unicorn.jpg'),
   new RandomPicture('usb', 'img/usb.gif'),
   new RandomPicture('waterCan', 'img/waterCan.jpg'),
   new RandomPicture('wineGlass', 'img/wineGlass.jpg')
 ];
 
-// for(var i = 0; i < stuff.length; i++){
-//   // console.log(stuff[i].name);
-//   // console.log(stuff[i].src);
-// }
-
 function findRandom(){
-  var tempArray = [];
-  while(tempArray.length < 3){
-    var rand = stuff[Math.floor(Math.random() * stuff.length)].src;
-    if(tempArray.indexOf(rand) === -1){
-      tempArray.push(rand);
-    }
-    // console.log(tempArray);
-  }
-  randomNumArray = tempArray;
+  // var tempArray = [];
+  // while(tempArray.length < 3){
+    // var rand = stuff[Math.floor(Math.random() * stuff.length)].src;
+  return(Math.floor(Math.random() * stuff.length));
+//     if(tempArray.indexOf(rand) === -1){
+//       tempArray.push(rand);
+//     }
+//     // console.log(tempArray);
+//   }
+//   randomNumArray = tempArray;
 }
-findRandom();
-console.log(randomNumArray);
 
-var pictureBlockEl = document.getElementById('productPictures');
-function postPictures(){
-  var pictureEl = document.createElement('h1');
-  pictureBlockEl.appendChild(pictureEl);
+function displayPics(){
+  var leftIndex = findRandom();
+  var centerIndex = findRandom();
+  var rightIndex = findRandom();
 
-  for(var i = 0; i < randomNumArray.length; i++){
-    var addPic = document.createElement('img');
-    addPic.setAttribute('src', randomNumArray[i]);
-    pictureEl.appendChild(addPic);
+  // console.log('starting left index', leftIndex);
+  // console.log('starting center index', centerIndex);
+  // console.log('starting right index', rightIndex);
 
-    console.log(typeof(randomNumArray[i]));
+  while(currentlyShowing.includes(leftIndex)){
+    leftIndex = findRandom();
+    // console.log('new left', leftIndex);
   }
-}
+  while(centerIndex === leftIndex || currentlyShowing.includes(centerIndex)){
+    centerIndex = findRandom();
+    // console.log('new center', centerIndex);
+  }
+  while(rightIndex === leftIndex || rightIndex === centerIndex || currentlyShowing.includes(rightIndex)){
+    rightIndex = findRandom();
+    // console.log('new right', rightIndex);
+
+  }
+  // console.log(leftIndex, centerIndex, rightIndex);
+  stuff[leftIndex].views ++;
+  stuff[centerIndex].views ++;
+  stuff[rightIndex].views ++;
+  // console.log(stuff[3].views, stuff[3].name);
+
+  pictureBlockEl.removeChild(left);
```

Comment:
It's probably cleaner to separate the part where you generate the random numbers into one function, and the part where you draw each of the images into another function.